src/nvim/menu.c

```diff
@@ -1308,6 +1308,9 @@ void ex_emenu(exarg_T *eap)
      * for exclusive mode */
     if (*p_sel == 'e' && gchar_cursor() != NUL)
       ++curwin->w_cursor.col;
+  } else if((State & CMDLINE)) {
```

Comment:
can you remove the extra parens? clint will also complain about the lack of a space in `if(`.